src/main/java/com/mercadopago/client/order/OrderClient.java

```diff
@@ -345,6 +346,80 @@ public OrderTransaction deleteTransaction(String orderId, String transactionId)
         return this.deleteTransaction(orderId, transactionId, null);
     }
 
+    /**
+     * Method responsible for creates a partial refunds for payment transactions
+     *
+     * @param orderId The ID of the order for which the refund is created
+     * @param requestOptions Metadata to customize the request
+     * @return The response for the order transaction
+     * @throws MPException    an error if the request fails
+     * @throws MPApiException an error if the request fails
+     */
+    public OrderTransaction refundPartial(String orderId, OrderRefundRequest request, MPRequestOptions requestOptions) throws MPException, MPApiException {
+        LOGGER.info("Sending order transaction intent request");
+
+        validateOrderID(orderId);
+
+        MPRequest mpRequest = MPRequest.builder()
+                .uri(String.format(URL_REFUND, orderId))
+                .method(HttpMethod.POST)
+                .payload(Serializer.serializeToJson(request))
+                .build();
+
+        MPResponse response = send(mpRequest, requestOptions);
+
+        OrderTransaction order = Serializer.deserializeFromJson(OrderTransaction.class, response.getContent());
+        order.setResponse(response);
+        return order;
+    }
+
+    /**
+     * Method responsible for a partial refund for payment transactions
+     * @param orderId        The ID of the order for which the refund is created
+     * @return The response for the order transaction
+     * @throws MPException    an error if the request fails
+     * @throws MPApiException an error if the request fails
+     */
+    public OrderTransaction refundPartial(String orderId, OrderRefundRequest request) throws MPException, MPApiException {
+        return this.refundPartial(orderId, request, null);
+    }
+
+    /**
+     * Method responsible for creates a total refunds for payment transactions without body
+     *
+     * @param orderId The ID of the order for which the refund is created
+     * @param requestOptions Metadata to customize the request
+     * @return The response for the order transaction
+     * @throws MPException    an error if the request fails
+     * @throws MPApiException an error if the request fails
+     */
+    public OrderTransaction refundTotal(String orderId, MPRequestOptions requestOptions) throws MPException, MPApiException {
```

Comment:
Aqui deve retornar um OrderRefund, essa deve ser atualizada com a nova versão do Swagger
```suggestion
    public OrderRefund refundTotal(String orderId, MPRequestOptions requestOptions) throws MPException, MPApiException {
```